### DISCUSSION TITLE
Javascript URL block

### DISCUSSION THREAD

**Discussion by creativename420** [3kh0/ext-remover#202] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my school blocked javascript:// urls, any ideas?

**Comment by Fallenmoon8080** [comment, marked as answer]
You're kinda boned.

**Reply by Endernates** [reply]
the real question is why would your school block that

**Reply by ghost** [reply]
@Endernates they know bookmarklets are a way to bypass
